docs/int/quickstart/advanced/quickstart-sdk.md

```diff
@@ -88,21 +90,48 @@ Once the Obol-API returns a `configHash` string from the `createClusterDefinitio
 
 1. Operators navigate to `https://<NETWORK_NAME_HERE>.launchpad.obol.tech/dv?configHash=<CONFIG_HASH_HERE>` and complete the [run a DV with others](../group/quickstart-group-operator.md) flow.
 1. Once the DKG is complete, and operators are using the `--publish` flag, the created cluster details will be posted to the Obol API
-1. The creator will be able to retrieve this data with `obol.getClusterLock(configHash)`, to use for activating the newly created validator. 
+1. The creator will be able to retrieve this data with `obol.getClusterLock(configHash)`, to use for activating the newly created validator.
 
 ## Retrieve the created Distributed Validators using the SDK
 
-Once the DKG is complete, the proposer of the cluster can retrieve key data such as the validator public keys and their associated deposit data messages. 
+Once the DKG is complete, the proposer of the cluster can retrieve key data such as the validator public keys and their associated deposit data messages.
 
 ```js
 const clusterLock = await obol.getClusterLock(configHash);
 ```
 
-Reference lock files can be found [here](https://github.com/ObolNetwork/charon/tree/main/cluster/testdata). 
+Reference lock files can be found [here](https://github.com/ObolNetwork/charon/tree/main/cluster/testdata).
 
 ## Activate the DVs using the deposit contract
 
-This guide will in future cover activating the deposit data from a lock file.
+In order to activate cluster validator, the cluster operator can retrieve the validator associated deposit data and call deposit from the deposit contract.
+
+```js
+const WEI_FACTOR = 10 ** 18;
+
+const validatorDepositData =
+  clusterLock.distributed_validators[validatorIndex].deposit_data;
+
+const depositContract = new ethers.Contract(
+  DEPOSIT_CONTRACT_ADDRESS, // 0x00000000219ab540356cBB839Cbe05303d7705Fa for Mainnet, 0xff50ed3d0ec03aC01D4C79aAd74928BFF48a7b2b for Goerli
+  depositContractABI, // https://etherscan.io/address/0x00000000219ab540356cBB839Cbe05303d7705Fa#code for Mainnet, and replace the address for Goerli
+  signer
+);
+
+const principal = BigNumber.from(validatorDepositData.amount); // 32 ethers
+
+const TX_VALUE = principal.mul(WEI_FACTOR);
```

Comment:
Have you tested this? I would have thought principal coming from our API ends up being `"32000000000"`. So multiplying by a further 18 decimals leads to too many decimals and too high a value?

I would imagine we multiply by `10 ** 9` to get the outcome we want. 

Here's a calculator to help test https://eth-converter.com/

Reply:
You are right. Fixing it